test/StdUtils.t.sol

```diff
@@ -90,7 +90,7 @@ contract StdUtilsTest is Test {
     }
 
     function testAssumeNoPrecompilesL1(address addr) external {
-        assumeNoPrecompiles(addr, stdChains.Mainnet.chainId);
+        assumeNoPrecompiles(addr, stdChains["mainnet"].chainId);
         assertTrue(addr < address(1) || addr > address(9));
     }
```

Comment:
This should be in `StdCheats.t.sol` and it should test Optimism, for example, because it didn't catch the `-` vs `_` change above.

Reply:
Done in 3212794